fe/fe-core/src/main/java/com/starrocks/sql/parser/StarRocks.g4

```diff
@@ -239,6 +235,11 @@ showAnalyzeStatement
     : SHOW ANALYZE
     ;
 
+// ------------------------------------------- Other Statement ---------------------------------------------------------
+showDatabasesStatement
```

Comment:
this has not been used now, am i right?

Reply:
showDatabasesStatement has been used now, but 'from catalog_name' won't make effects.It's downward compatible.